demo/monolayer_growth/src/cell_cell_force.cc

```diff
@@ -0,0 +1,72 @@
+// -----------------------------------------------------------------------------
+//
+// Copyright (C) 2021 CERN & University of Surrey for the benefit of the
+// BioDynaMo collaboration. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// See the LICENSE file distributed with this work for details.
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// -----------------------------------------------------------------------------
+
+#include "cell_cell_force.h"
+#include "sim_param.h"
+
+namespace bdm {
+
+Real4 CellCellForce::Calculate(const Agent* lhs, const Agent* rhs) const {
```

Comment:
Can we maybe add a comment to describe the motivation for replacing the force and what has changed compared to the original "bdm native" implementation?
```
/// Custom force. Changed adhesive and repulsive parameter compared to standard force.
Real4 CellCellForce::[...]
```